util/prep-bazel-airgapped-build.sh

```diff
@@ -154,7 +154,8 @@ if [[ ${AIRGAPPED_DIR_CONTENTS} == "ALL" || \
     @rust_freebsd_x86_64_toolchains//... \
     @rust_linux_aarch64_toolchains//... \
     @rust_linux_x86_64_toolchains//... \
-    @rust_windows_x86_64_toolchains//...
+    @rust_windows_x86_64_toolchains//... \
+    //quality/...
```

Comment:
@timothytrippel 
i think both @eunchan and i agree this doesn't seem like the right fix..but it works for now..

Reply:
actually i have no idea what i'm talking about hehe.